app/services/sso/basic_with_hidden.rb

```diff
@@ -0,0 +1,11 @@
+module SSO
+  class BasicWithHidden < Basic
+    def available?
+      controller.api_request? && http_auth_set? && controller.is_a?(Katello::Api::Rhsm::CandlepinProxiesController)
+    end
+
+    def current_user
+      User.unscoped.find_by_login(self.user)
```

Comment:
Is it safe to open the rhsm actions for all users? We could allow access only for users generated by the plugin.